interfaces/builtin/network_manager.go

```diff
@@ -67,6 +67,29 @@ network packet,
 
 /dev/rfkill rw,
 
+# Needed for modem connections using PPP
+/usr/sbin/pppd ix,
+/etc/ppp/** rwix,
+/dev/ppp rw,
+/dev/tty[^0-9]* rw,
+/run/udev/data/* r,
+/proc/sys/kernel/modprobe r,
```

Comment:
It would be usable by both ofono and modem-manager and we don't need to write the same rules into both interfaces.